pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -85,10 +85,10 @@ func (p RESTStorageProvider) storage(version schema.GroupVersion, apiResourceCon
 	once := new(sync.Once)
 	var (
 		authorizationRuleResolver  rbacregistryvalidation.AuthorizationRuleResolver
-		rolesStorage               rest.StandardStorage
-		roleBindingsStorage        rest.StandardStorage
-		clusterRolesStorage        rest.StandardStorage
-		clusterRoleBindingsStorage rest.StandardStorage
+		rolesStorage               rolepolicybased.Registry
```

Comment:
From what I can see this interface is identical in each sub package. Why not define the interface centrally instead of 4 different times?